### DISCUSSION TITLE
Build for windows with OPUS

### DISCUSSION THREAD

**Discussion by frequencebanane** [baresip/baresip#2390] [Q&A]
Hello everyone :)
I'm not used to build app for windows so i'm little bit lost. How can I build baresip and include opus module ?
I need to first install cmake on windows is that right ? 

Best regards, 
Alexandre

**Comment by alfredh** [comment]
you need to use CMake and the right generator that is best for you.
Try this and see what happens:

```
$ cd baresip
$ cmake .
```
